src/i18n/translations.ts

```diff
@@ -66,7 +66,8 @@ export default {
         confirmMessage: 'Bitte bestätigen Sie, dass Sie alle Zeichnungen löschen möchten.'
       },
       StylingDrawer: {
-        pickColor: 'Farbschema bearbeiten',
+        openColorPalette: 'Farbpalette öffnen',
+        closeColorPalette: 'Farpalette schließen',
```

Comment:
```suggestion
        closeColorPalette: 'Farbpalette schließen',
```

Reply:
done in: [2c93f63](https://github.com/terrestris/shogun-gis-client/pull/1562/commits/2c93f63d742d1e0b3e3778b754180dccbc371a59)